cypress/mocks/migrated-from-pact.json

```diff
@@ -97,18 +97,6 @@
         "body": "{\"opg.poas.sirius\":{\"application\":{\"correspondentAddress\":{\"addressLine1\":\"Main Line Bungalow\",\"addressLine2\":\"Himmerton Lane\",\"addressLine3\":\"Sutton\",\"country\":\"GB\",\"postcode\":\"S24 7DJ\",\"town\":\"Scarsdale\"},\"correspondentFirstNames\":\"Heath\",\"correspondentLastName\":\"Enstad\",\"donorAddress\":{\"addressLine1\":\"Apartment 24\",\"addressLine2\":\"Navigation Building\",\"addressLine3\":\"90 London Road\",\"country\":\"GB\",\"postcode\":\"B15 4TA\",\"town\":\"Birmingham\"},\"donorDob\":\"27/05/1978\",\"donorEmail\":\"zswanberg@host.example\",\"donorFirstNames\":\"Zoraida\",\"donorLastName\":\"Swanberg\",\"donorPhone\":\"073456249524\"},\"caseSubtype\":\"pfa\",\"complaintCount\":1,\"createdDate\":\"31/08/2023\",\"investigationCount\":2,\"status\":\"Draft\",\"taskCount\":3,\"id\":900,\"warningCount\":4},\"uId\":\"M-9999-4567-AAAA\"}"
       }
     },
-    {
```

Comment:
Is this definitely not used by any of the other cypress tests?

Reply:
I'm fairly certain it isn't, only in the digital lpa cypress tests do we perform the API call `/lpa-api/v1/digital-lpas/M-DIGI-LPA3-3333`